packages/mesos/extra/dcos-mesos-slave.service

```diff
@@ -12,6 +12,7 @@ EnvironmentFile=/opt/mesosphere/environment
 EnvironmentFile=/opt/mesosphere/etc/mesos-slave-common
 EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-common-extras
 EnvironmentFile=/opt/mesosphere/etc/mesos-slave
+EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-extras
```

Comment:
Please consider that a downstream distro/operator may want to configure public agents to use a different principal for authenticating to the master, or use different certs/keys for encrypted communication.